public/css/proposal.css

```diff
@@ -0,0 +1,56 @@
+.proposal{
+  width: 70%;
+  margin: 50px auto;
+  text-align: left;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+ }
+ .proposal .user-info{
+  height: 100px;
+  display: flex;
+ }
+ 
+ .proposal .user-info .left-user-info,
+ .proposal .user-info .right-user-info{
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+ }
+ .proposal .user-info .left-user-info{
+  width: 50px;
+  height: 50px;
+  background: #333;
+  border-radius: 50%;
+  margin: auto 0;
+ }
+ .proposal .user-info .right-user-info h2{
+  flex-direction: row;
```

Comment:
@AhmedAl-Almi Is this flex-direction a typo?  I think flex-direction won't do anything for this h2 (since it has no child elements, and isn't display: flex)